.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: generic
+
+env:
+    global:
+        - PYTHONIOENCODING=UTF8
+        - MPLBACKEND=Agg
+
+matrix:
+  include:
+    - os: linux
+      language: generic
+      env:
+        - PYTHON_VERSION=3.6
+        - CONDA=true
+
+    - os: linux
+      language: generic
+      env:
+        - PYTHON_VERSION=3.7
+        - CONDA=true
+
+before_install:
+
+    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - . $HOME/miniconda/etc/profile.d/conda.sh
+    - hash -r
+    - conda config --set always_yes yes --set changeps1 no
+    - conda update -q conda  # get latest conda version
+    - conda info -a # Useful for debugging any issues with conda
+
+install:
+    - conda env create -f protopipe_environment.yml
+    - conda activate protopipe
+    - pip install travis-sphinx codecov pytest-cov
+    - python setup.py install
+
+script:
+    # - pytest --cov=protopipe
```

Comment:
no tests?

Reply:
They will come in a separate PR, I am working on an initial set.
I just hope that afterward I just de-comment this line and everything magically works.